README.md

```diff
@@ -59,87 +66,82 @@ The following instructions detail the process of getting the software, building
 <a name="autobuild"></a>
 ### Automated build script
 
-For Ubuntu 16.10 and MacOS Sierra, there is an automated build script that can install all dependencies and builds EOS.
+Supported Operating Systems:  
+1. Ubuntu 16.04 and higher.  
+2. MacOS Darwin 10.12 and higher.  
+3. Fedora 25 and higher.  
 
-It is called eosio-build.sh with the following inputs.
-- architecture [ubuntu|darwin]
-- optional mode [full|build]
+For Ubuntu, MacOS and Fedora there is an automated build script that can install all dependencies and builds EOS.
+We are working on supporting Centos, Amazon Linux & Red Hat in future releases.
 
-The second optional input can be `full` or `build` where `full` implies that it installs dependencies and builds eos. If you omit this input then the build script installs dependencies and then builds eos.
+It is called eosio_build.sh
 
 ```bash
-./eosio-build.sh <architecture> <optional mode>
+cd eos
+./eosio_build.sh
 ```
-Choose whether you will be building for a local testnet or for the public testnet and jump to the appropriate section below.  Clone the EOS repository recursively as described and run eosio-build.sh located in the root `eos` folder.
+Choose whether you will be building for a local testnet or for the public testnet and jump to the appropriate section below.  Clone the EOS repository recursively as described and run eosio_build.sh located in the root `eos` folder.
 
 :warning: **As of February 2018, `master` is under heavy development and is not suitable for experimentation.** :warning:
 
 We strongly recommend following the instructions for building the public testnet version for [Ubuntu](#autoubuntupublic) or [Mac OS X](#automacpublic). `master` is in pieces on the garage floor while we rebuild this hotrod. This notice will be removed when `master` is usable again. Your patience is appreciated.
 
 <a name="autoubuntulocal"></a>
-#### :no_entry: Clean install Ubuntu 16.10 for a local testnet :no_entry:
+#### Clean install Linux (Ubuntu & Fedora) for a local testnet
 
 ```bash
 git clone https://github.com/eosio/eos --recursive
 
 cd eos
-./eosio-build.sh ubuntu
+./eosio_build.sh
 ```
 
 Now you can proceed to the next step - [Creating and launching a single-node testnet](#singlenode)
 
 <a name="autoubuntupublic"></a>
-#### Clean install Ubuntu 16.10 for the public testnet
+#### Clean install Linux (Ubuntu & Fedora) for the public testnet
 
 ```bash
 git clone https://github.com/eosio/eos --recursive
 
 cd eos
-git checkout dawn-2.x
-./eosio-build.sh ubuntu
+git checkout DAWN-2018-01-25
+./eosio_build.sh
 ```
 
 Now you can proceed to the next step - [Running a node on the public testnet](#publictestnet)
 
 <a name="automaclocal"></a>
-#### :no_entry: MacOS Sierra for a local testnet :no_entry:
```

Comment:
Restore no_entry emojis.

Reply:
Note: Corey specifically requested these to be removed. I put them back as per your request.